package.json

```diff
@@ -75,30 +60,26 @@
     "jsdoc": "3.4.0",
     "jshint": "2.9.1",
     "json-loader": "0.5.4",
-    "load-grunt-config": "0.17.2",
-    "load-grunt-tasks": "3.2.0",
     "memdown": "1.0.0",
+    "mocha": "2.4.5",
```

Comment:
@cdaringe @zombieprocess When I review Jamin's fixes I'll add mocha removal as well. The jscs/jshint removal is due to our incoming change to eslint (maybe I jumped to gun a bit, but soon!)